src/components/notebook/macro-panel.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "@testing-library/jest-dom";
+import { DndProvider } from "react-dnd";
+import { TouchBackend } from "react-dnd-touch-backend";
```

Comment:
Note that if you're not actually testing the drag/drop functionality, you can mock it as I did in `macro-animal-row.test.tsx`, something like the following:
```typescript
jest.mock("react-dnd", () => ({
  useDrop: () => [{ isOver: false }, () => null]
}));
```

Reply:
Fixed.